cmd/node/node.go

```diff
@@ -344,9 +351,8 @@ func (app *SpacemeshApp) setupGenesis(state *state.TransactionProcessor, msh *me
 	for id, acc := range conf.InitialAccounts {
 		bytes := util.FromHex(id)
 		if len(bytes) == 0 {
-			// todo: should we panic here?
 			app.log.With().Error("cannot read config entry for genesis account", log.String("acct_id", id))
-			continue
+			return errors.New("id in initial accounts is empty")
```

Comment:
It's a bad idea to log one errmsg here and return another one. Probably best to just return the existing errmsg (which is more informative) and print it at a higher level.